kumascript/src/api/util.js

```diff
@@ -141,9 +141,21 @@ class HTMLTool {
     // so let's simplify this as well as make it much faster.
     const sectionStart = $(`#${cssesc(sectionID, { isIdentifier: true })}`);
     if (!sectionStart.length) {
-      throw new KumascriptError(
-        `unable to find an HTML element with an "id" of "${sectionID}" within ${this.pathDescription}`
+      let errorMessage = `unable to find an HTML element with an "id" of "${sectionID}" within ${this.pathDescription}`;
+      const hasMoreThanAscii = [...sectionID].some(
+        (char) => char.charCodeAt(0) > 127
       );
+      if (hasMoreThanAscii) {
+        errorMessage +=
+          " Hint! Remove any non-ASCII characters and (in the ID) and it will probably work.";
+        const cleanedSectionID = [...sectionID]
+          .filter((char) => char.charCodeAt(0) <= 127)
+          .join("");
+        errorMessage +=
+          ` Try changing it to 'id="${cleanedSectionID}"' ` +
+          `and EmbedLiveSample('${cleanedSectionID}', ...)`;
```

Comment:
Can I suggest this instead? I think it reads a little better.
```suggestion
        const cleanedSectionID = [...sectionID]
          .filter((char) => char.charCodeAt(0) <= 127)
          .join("");
        errorMessage +=
          ' -- hint! removing all non-ASCII characters in the "id" may fix this, so try ' +
          `'id="${cleanedSectionID}"' and 'EmbedLiveSample("${cleanedSectionID}", ...)'`;
```

And here's an example of how it would look:

<img width="1016" alt="image" src="https://user-images.githubusercontent.com/3743693/117184224-56038f80-ad8d-11eb-867e-74983def6bdb.png">